packages/tokens/src/semantic/index.ts

```diff
@@ -0,0 +1,411 @@
+export type SemanticTokens = {
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32104&commitId=4e9698e11c31a70947ccd8c6b8e4d3398b337ac1&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|4| Changed|
|Avatar Converged.badgeMask.normal.chromium.png|6| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>